content/072-relaxed-dedented-case-syntax.md

```diff
@@ -0,0 +1,275 @@
+---
+layout: sip
+number: 72
+permalink: /sips/:number.html
+redirect_from:
+  - /sips/:number
+  - /sips/:title.html
+stage: pre-sip
+status: under-review
+title: Relaxed Dedented Case Syntax
+---
+
+**By: Li Haoyi**
+
+## History
+
+| Date              | Version            |
+|-------------------|--------------------|
+| December 31st 2025| Initial Draft      |
+
+## Summary
+
+This SIP proposes to allow non-indented `case` clauses in non-match contexts, such
+as partial function literals and catch blocks:
+
+```scala
+xs.map:
+case (a, b) => a + b
+case (a, b, c) => a + b + c
+
+val pf: PartialFunction[String, Int] =
+case "foo" => 1
+case "bar" => 2
```

Comment:
For the PR, I'll look at 2.1.4 in

https://dotty.epfl.ch/docs/reference/other-new-features/indentation.html#optional-braces

where `case` modifies the condition in 2.1.2. It's saying, wherever you could have indented, you can have a `case` block at same indentation.  This example won't work yet.